v63003/u_inref/gtm8680.csh

```diff
@@ -0,0 +1,32 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests YDB does not experience significant slowdown with large number locks
+#
+$gtm_tst/com/dbcreate.csh mumps 1 -lock_space=60000>& a.out
```

Comment:
Add a comment that the lockspace of 60000 is needed to ensure we never call the garbage collector (which we can if we have a small lock space) as that will clean up the unused lock names from shared memory thereby speeding up future lock commands even in the version that does not have the GTM-8680 fix.

Reply:
This comment does not seem to have been incorporated.

Reply:
This comment still does not seem to have been incorporated.